src/components/LocationBias.tsx

```diff
@@ -11,15 +11,15 @@ import LoadingIndicator from '../icons/LoadingIndicator';
  * @public
  */
 export interface LocationBiasCssClasses {
-  container?: string,
+  locationBiasContainer?: string,
   location?: string,
   source?: string,
   button?: string,
   loadingIndicatorContainer?: string
 }
 
 const builtInCssClasses: Readonly<LocationBiasCssClasses> = {
-  container: 'text-sm text-neutral text-center flex justify-center items-center',
+  locationBiasContainer: 'text-sm text-neutral text-center justify-center items-center flex flex-col lg:flex-row',
```

Comment:
would `md:flex-row` work here instead of `lg:flex-row`?

Reply:
if you use md, there's this small awkward range where it looks scrunched like the original issue.   lg seems to work well for names with reasonable length
![image](https://user-images.githubusercontent.com/59857107/177630050-b0abf60e-bb9c-4fb7-be16-9f9a46b97e6f.png)

Reply:
ok, `lg` is fine then!